modules/system/classes/PluginManager.php

```diff
@@ -631,6 +632,7 @@ public function disablePlugin($id, $isUser = false)
         if ($pluginObj = $this->findByIdentifier($code)) {
             $pluginObj->disabled = true;
         }
+        Event::fire('system.plugins.afterDisable', [$code]);
```

Comment:
@mjauvin almost forgot, we need the inline event docblocks on these

Reply:
done @LukeTowers !